ttnn/cpp/ttnn/operations/eltwise/unary/unary_composite.hpp

```diff
@@ -97,67 +97,53 @@ struct ExecuteUnaryCompositeOp
-struct ExecuteUnaryCompositeOpWithScaleShift
-{
-    static ttnn::Tensor execute_on_worker_thread(
-        const Tensor& input_tensor,
-        float scale,
-        float shift,
-        const std::optional<MemoryConfig>& memory_config = std::nullopt)
-        {
-            auto op_type = get_function_type2<unary_comp_op_type>();
-            return op_type(input_tensor, scale, shift, memory_config);
+struct ExecuteUnaryCompositeOpWith2Float {
+
+    //Type 1: 1 inputs, 2 float
+    static ttnn::Tensor execute_on_main_thread(
+        const Tensor &input_tensor,
+        float param1,
+        float param2,
+        const std::optional<MemoryConfig> &memory_config = std::nullopt) {
+        auto op_type = get_function_type_2float<unary_comp_op_type>();
+        auto output_memory_config = memory_config.value_or(input_tensor.memory_config());
+        return op_type(input_tensor, param1, param2, output_memory_config);
         }
-
 };
 
-template <UnaryCompositeOpType unary_comp_op_type>
-struct ExecuteUnaryCompositeOpWithLowHigh
-{
-    static ttnn::Tensor execute_on_worker_thread(
-        const Tensor& input_tensor,
-        float low,
-        float high,
-        const std::optional<MemoryConfig>& memory_config = std::nullopt)
-        {
-            auto op_type = get_function_type3<unary_comp_op_type>();
-            return op_type(input_tensor, low, high, memory_config);
-        }
-
-};
 
+//OpHandler_one_int : get_function_type_1int
 template <UnaryCompositeOpType unary_comp_op_type>
-struct ExecuteUnaryCompositeOpWithScaleAlpha
-{
-    static ttnn::Tensor execute_on_worker_thread(
-        const Tensor& input_tensor,
-        float scale,
-        float alpha,
-        const std::optional<MemoryConfig>& memory_config = std::nullopt)
-        {
-            auto op_type = get_function_type4<unary_comp_op_type>();
-            return op_type(input_tensor, scale, alpha, memory_config);
+struct ExecuteUnaryCompositeOpWith1int {
+
+    //Type 1: 1 inputs, 2 float
+    static ttnn::Tensor execute_on_main_thread(
+        const Tensor &input_tensor,
+        int32_t param1,
+        const std::optional<MemoryConfig> &memory_config = std::nullopt) {
+        auto op_type = get_function_type_1int<unary_comp_op_type>();
+        auto output_memory_config = memory_config.value_or(input_tensor.memory_config());
+        return op_type(input_tensor, param1, output_memory_config);
         }
-
 };
 
 template <UnaryCompositeOpType unary_comp_op_type>
-struct ExecuteUnaryCompositeOpWithThresholdValue
-{
-    static ttnn::Tensor execute_on_worker_thread(
-        const Tensor& input_tensor,
-        float threshold,
-        float value,
-        const std::optional<MemoryConfig>& memory_config = std::nullopt)
-        {
-            auto op_type = get_function_type5<unary_comp_op_type>();
-            return op_type(input_tensor, threshold, value, memory_config);
+struct ExecuteUnaryCompositeOpWith1intBool {
```

Comment:
if this is used for a specific op only, rename struct ExecuteUnaryCompositeOpWith1intBool to opname in camel case